Makefile

```diff
@@ -391,6 +391,32 @@ vet:
 gen-helm-docs: ## Generate Helm docs
 	@docker run -v ${PWD}:/work -w /work jnorwood/helm-docs:v1.6.0 -s file
 
+.PHONY: gen-helm-crds
+gen-helm-crds: kyverno-crd report-crd ## Generate Helm CRDs template
+	{ \
+		echo "{{- if .Values.installCRDs }}"; \
+		cat ./config/crds/kyverno.io_clusterpolicies.yaml \
+			./config/crds/kyverno.io_clusterreportchangerequests.yaml \
+			./config/crds/kyverno.io_generaterequests.yaml \
+			./config/crds/kyverno.io_policies.yaml \
+			./config/crds/kyverno.io_reportchangerequests.yaml \
+			./config/crds/wgpolicyk8s.io_clusterpolicyreports.yaml \
+			./config/crds/wgpolicyk8s.io_policyreports.yaml; \
+		echo "{{- end }}"; \
+	} \
+	> charts/kyverno/templates/crds.yaml
```

Comment:
We update Helm CRDs via `kustomize-crd` : https://github.com/kyverno/kyverno/blob/4c1a8336b0267204bacfd995904589601364ebee/Makefile#L289

Should we replace it with this new command?

Reply:
Nice catch, i didn't see this Makefile target.
It's better to reuse the same process as the release.
I'll check that asap.

Reply:
Indeed, it's better to reuse `kustomize-crd` target. I updated the PR.

Reply:
I would expect that manifests are up to date when checking out the main branch, shouldn't we add a step in the CI to check release manifests are up to date ?

Reply:
> I would expect that manifests are up to date when checking out the main branch, shouldn't we add a step in the CI to check release manifests are up to date ?

Yes, ideally we want to automate this. Not sure what's the best way to do it.

Reply:
>We could verify it and fail the CI if it's not up to date.

Yes, can we add this to CI `verify-helm` and fail the test if it's not up-to-date?

Reply:
Yeah, I’ll do that in another pr.